pkg/client/redhatsso/client.go

```diff
@@ -108,6 +108,7 @@ func (c *rhSSOClient) GetToken() (string, error) {
 	client := &http.Client{}
 	parameters := url.Values{}
 	parameters.Set("grant_type", "client_credentials")
+	parameters.Set("scope", "api.iam.service_accounts")
```

Comment:
Lets add the scope name as a constant, so that it can be maintained in one place. WDYT?

Reply:
yes, I will update once the changes are ready to be tested against sso env